feat(docs): split Okta SCIM docs from SAML page (#3899)
getsentry/sentry-docs: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx

---
title: "Okta SCIM Provisioning"
sidebar_order: 3
description: "Set up Okta's SCIM Integration for Member and Team Provisioning"
---
<Note>

This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.

If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

### Supported Features

> - Create users

Reviewer: Add a space after the heading
```suggestion

- Create users
```

- Deactivate users
- Push groups
- Import groups

> ### Requirements
> - You must be on a Sentry Business Plan or above.
> - You must have configured SAML SSO for okta. See [documentation here](/product/accounts/sso/okta-sso/), or the Okta sign-on tab in your Sentry Okta application for configuring SAML.
> - In your Sign On configuration, **You must select Email for the Application username format** on the Sign On application tab in Okta.

Reviewer: ```suggestion
### Requirements

Okta Skim provisioning requires:
- A subscription to Sentry Business Plan or higher.
- Configuration of SAML SSO for Okta[as documented here](/product/accounts/sso/okta-sso/), or use the Okta sign-on tab in your Sentry Okta application to configure SAML.
- Selection of **You must select Email for the Application username format** in the Sign On application tab in Okta.
```

   ![Okta Test SCIM Credentials](scim-okta-test.png)
   ![SCIM Verified Sucessfully](scim-okta-verified.png)

1. Select "Save" to be directed to SCIM App settings:
   ![Okta Save SCIM Settings](scim-okta-save.png)

1. On the Provisioning page, select "To App", then "edit":
   ![Okta Save SCIM Settings](scim-okta-edit.png)

1. Enable both "Create Users" and "Deactivate Users", then "Save" your changes.

As a result of these changes, users who are assigned will be sent an invitation email. When a user is un-assigned, their membership object in Sentry will be deleted.

You can use "Push Groups" to sync and assign groups in Okta; they will be reflected in Sentry teams.

### Known Issues / Troubleshooting
- Sentry does not currently support setting any User attributes other than `userName` and `active`
- The User `name` attribute will contain fields firstName and lastName with the values of `N/A`. Sentry's SCIM API does not currently support these fields but returns them for compatibility purposes.
- Setting `active` to `false` on a User will delete the organization member record associated with the user.
- The only filter operation supported for resources is `eq`.

> - When provisioning a new team, Sentry will normalize and convert the team `displayName` uppercase to lowercase, and spaces to dashes.

Reviewer: ```suggestion
- When provisioning a new team, Sentry will both normalize and convert the team `displayName` uppercase to lowercase, and convert spaces to dashes.
```

- The GET /Groups endpoint will return a maximum of 10000 members in a group.

> - See [SCIM API documentation](/api/scim/) for more information.

Reviewer: If this is related to the GET /Groups endpoint, combine these two bullets.
